doc/api/http.md

```diff
@@ -79,7 +92,11 @@ http.get(options, (res) => {
 });
 ```
 
-Alternatively, you could just opt out of pooling entirely using
+You may also use an agent for an individual request. By providing
+`{agent: false}` as an option to the `http.get()` or `http.request()`
+functions, a one-time use `Agent` with default options and no connection
+pooling will be used for the client connection.
```

Comment:
I think the phrase 'and no connection pooling will be used for the client connection' is a little confusing when talking about a single socket. Perhaps just remove it altogether, since that is basically what was done in the description for the `agent` option in `http.request()`.